# Array of detections from perception node
std_msgs/Header header

Detection[] detections
